editor/doc_tools.cpp

```diff
@@ -256,8 +257,16 @@ void DocTools::generate(bool p_basic_types) {
 
 		List<PropertyInfo> properties;
 		List<PropertyInfo> own_properties;
-		if (name == "ProjectSettings") {
-			//special case for project settings, so settings can be documented
+
+		if (name == "EditorSettings") {
+			// Special case for editor settings, so they can be documented.
+			// Force the use of default editor settings to prevent the user's settings
+			// (and list of projects) from being included the list.
+			EditorSettings::create(true);
+			EditorSettings::get_singleton()->get_property_list(&properties);
+			own_properties = properties;
```

Comment:
Settings which are defined inside editor constructors (e.g. EditorNode `EDITOR_DEF("run/output/always_clear_output_on_play", true);`) will not be picked up on since the editor is not instantiated.